wisp-ssl/README.md

```diff
@@ -0,0 +1,23 @@
+# wisp-ssl
+
+Loads and parses SSL certificates into
+a [`KeyStore`](https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/security/KeyStore.html)
+.
```

Comment:
nit: move dot to previous line?

Reply:
ah, it auto-formatted. it should get hidden in markdown, but can change later.